qiskit_addon_sqd/fermion.py

```diff
@@ -66,6 +79,316 @@ def load(cls, filename):
         with np.load(filename) as data:
             return cls(data["amplitudes"], data["ci_strs_a"], data["ci_strs_b"])
 
+    def rdm(self, rank: int = 1, spin_summed: bool = False) -> np.ndarray:
```

Comment:
we could prefix this with `generate_` to make it more clear a calculation is going to happen every time you call it

Reply:
Oh, the way I see things, the fact that it's not a [property](https://docs.python.org/3/library/functions.html#property) already suggests that a calculation is going to happen every time. I prefer not to add the `generate_` prefix because it seems redundant to me, but I'll add it if you insist.